src/plugins/trigger/trigger-buffer.h

```diff
@@ -22,7 +22,7 @@
 
 #define TRIGGER_BUFFER_NAME "monitor"
 
-struct t_gui_buffer *trigger_buffer;
```

Comment:
Probably my branch launches from a previous commit to the one that incorporates those changes.  Let me revise it this evening, as now I'm at work.